specs/ClearBrowsingData.md

```diff
@@ -100,79 +100,86 @@ interface ICoreWebView2ClearBrowsingDataCompletedHandler;
 
 /// Specifies the datatype for the
 /// `ICoreWebView2Profile::ClearBrowsingData` method.
-[v1_enum] 
+[v1_enum]
 typedef enum COREWEBVIEW2_BROWSING_DATA_KINDS {
+  /// [Deprecated] This enum has already been deprecated in the current implementation, 
+  /// and don't use it any more.
   /// Specifies data stored by the AppCache DOM feature.
-  COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE = 1 << 0,
+  /// COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE = 1 << 0,
```

Comment:
The enum value needs to continue to exist. It cannot be commented out. It will be documented as deprecated and we'll see if we can add support for more deprecation mechanisms (whatever .NET & WinRT support) in the future.

Reply:
Will do.

Reply:
Per discussion, you have pointed out that we never shipped this enum value even as experimental so we can just remove it from here

Reply:
I agree.